apps/src/code-studio/pd/application/teacher/ProfessionalLearningProgramRequirements.jsx

```diff
@@ -183,17 +184,14 @@ const ProfessionalLearningProgramRequirements = props => {
             workshop information.
           </p>
           <p>
-            Refresh the page to try again. If this persists, please
-            contact&nbsp;
+            Refresh the page to try again. If this persists, please contact{' '}
```

Comment:
TIL that &nbsp and {' '} have different character codes https://stackoverflow.com/questions/1357078/whats-the-difference-between-nbsp-and